EFCore.VisualBasic.Test/Migrations/Design/VisualBasicMigrationsGeneratorTests.vb

```diff
@@ -404,16 +403,16 @@ End Namespace
                        sqlOp,
                         New AlterColumnOperation With
                         {
-                            .Name = "C2",
-                            .Table = "T1",
-                            .ClrType = GetType(Storage.Database),  ''''''''''''''''''''''''''''''''''''''''''''''''''' ???????????
-                            .OldColumn = New ColumnOperation With {.ClrType = GetType([Property])}
+                             .Name = "C2",
+                             .Table = "T1",
+                             .ClrType = GetType(Storage.Database),  ''''''''''''''''''''''''''''''''''''''''''''''''''' ???????????
```

Comment:
You can probably remove this ''''? stuff  I have forgotten to remove before commiting 😄